CHANGELOG.md

```diff
@@ -72,6 +72,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Fixed
 
 - Correct the release date of `0.20.0` to `2024-04-25`.
+- Fixed `Int` type precision error that occurs when graphql API have instances
+  that are greater than `i32` (`u32`,`u64`,`isize`,`usize`,`i64`) for
+  `Round`,`ClusterTrend`,`TopMultiMaps`,`TopElementCountsByColumn`,
+  `ElementCount`,`StructuredClusteringTemplate`.
```

Comment:
Either `Changed` or `Fixed` would work, but I think we mainly need to notify users about the API changes, so `Changed` might be slightly better.

Additionally, if there are no other areas where this change applies, it would be simpler to mention something like 'all corresponding APIs' or whatever wording you feel is appropriate, instead of listing every single item.

Reply:
I see the approach. I will make the appropriate changes.

Reply:
Done, changed to "all applicable APIs"